apps/jubmoji-quest/src/pages/quests/[id].tsx

```diff
@@ -49,7 +58,19 @@ export default function QuestDetailPage() {
           title={quest.name}
           description={quest.description}
           image={quest.imageLink || ""}
-        />
+        >
+          <div className="flex flex-col gap-4 mt-6">
+            <div className="flex flex-col">
+              <Card.Title className="!text-base text-left">Collect:</Card.Title>
+              <div className="flex gap-2">[COLLETED_ITEMS]</div>
```

Comment:
@kalidiagne Should we be showing the emojis here?

Reply:
@vb7401  yep! we don't have the collection list yet

Reply:
We can just use `quest.collectionCards` and fetch the emojis from the card indices, no?